src/render/canvas/canvas-renderer.ts

```diff
@@ -727,15 +727,14 @@ export class CanvasRenderer extends Renderer {
                 .forEach((shadow) => {
                     this.ctx.save();
                     const borderBoxArea = calculateBorderBoxPath(paint.curves);
-                    const maskOffset = shadow.inset ? 0 : MASK_OFFSET;
+                    const maskOffset = shadow.inset ? 0 : 1;
```

Comment:
This might be correct, but I find it suspicious that switching a value from 10000 to 1 is not having bad impacts somewhere. Are you sure that this cannot introduce bugs elsewhere?